### PR TITLE
Add manifold teams remove command to revoke invites or remove members

### PR DESCRIPTION
**Pull request by jeffandersen**
```
$ manifold teams invite jeff+test@manifold.co
✔ Select Team: awesomesauce (awesomesauce)
✔ Name: Jeff Andersen
✔ Select Role: read
An invite has been sent to Jeff Andersen <jeff+test@manifold.co>
```
```
$ manifold teams members
1 members and 1 invites
Use `manifold switch` to change to a different team

Name                 Email                        Role         Status
jeff                 jeff@manifold.co             admin        active
Jeff Andersen        jeff+test@manifold.co        read         pending
```
```
$ manifold teams remove jeff+test@manifold.co
Invite to Jeff Andersen <jeff+test@manifold.co> has been revoked
```

### CODE REVIEW

**Comment by jeffandersen** [comment]
I pushed a CHANGELOG update but Github hasn't caught up yet

File: cmd/teams.go [hunk 1]
```diff
@@ -47,6 +47,14 @@ func init() {
 				Action: middleware.Chain(middleware.EnsureSession, updateTeamCmd),
 			},
 			{
+				Name:      "remove",
+				ArgsUsage: "[email]",
+				Usage:     "Invite a user to join a team",
```
Comment (jbowes) [inline cmd/teams.go]: nopers

File: cmd/teams.go [hunk 2]
```diff
@@ -477,6 +485,101 @@ func selectTeam(ctx context.Context, teamName string, identityClient *client.Ide
 	return team, nil
 }
 
+func removeFromTeamCmd(cliCtx *cli.Context) error {
+	ctx := context.Background()
+	args := cliCtx.Args()
+
+	// Only allow args if exactly two are supplied
```
Comment (jbowes) [inline cmd/teams.go]: that's not whats going on here

File: cmd/teams.go [hunk 3]
```diff
@@ -477,6 +485,101 @@ func selectTeam(ctx context.Context, teamName string, identityClient *client.Ide
...
+		prompts.SpinStop()
+		if err != nil {
+			return cli.NewExitError(fmt.Sprintf("Failed to revoke invite: %s", err), -1)
+		}
+
+		fmt.Printf("Invite to %s <%s> has been revoked\n", name, email)
+		return nil
+	}
+
+	prompts.SpinStart("Revoking invite")
```
Comment (jbowes) [inline cmd/teams.go]: this isn't the invite

File: cmd/teams.go [hunk 4]
```diff
@@ -477,6 +485,98 @@ func selectTeam(ctx context.Context, teamName string, identityClient *client.Ide
 	return team, nil
 }
 
+func removeFromTeamCmd(cliCtx *cli.Context) error {
+	ctx := context.Background()
+	args := cliCtx.Args()
+
+	var err error
+	if len(args) > 0 {
+		email, err = optionalArgEmail(cliCtx, 0, "email")
```
Comment (enmand) [inline cmd/teams.go]: Missing `var email string` above?

Comment (jeffandersen) [reply cmd/teams.go]: Fixed!

**Pull request merged**